conf/zapiperf/cdot/9.8.0/volume.yaml

```diff
@@ -24,6 +24,10 @@ counters:
 
 plugins:
   - Volume
+  - Aggregator:
+    # plugin will create summary/average for each object
+    # any names after the object names will be treated as label names that will be added to instances
+    - node
```

Comment:
we need node level aggregation without any extra labels.
This node aggregation would aggregate this way:
- latency(read/write/average): weighted average
- data(read/write): sum
- ops(total): sum